### DISCUSSION TITLE
web:build: Type error: Already included file name

### DISCUSSION THREAD

**Discussion by hrutik7** [vercel/community#5086] [Help] [answered]
### Summary

while trying to deploying on vercel I am facing one error due to case sensitivity for that i used command `git config core.ignorecase false` and pushed the code but I am getting different error now 

![image](https://github.com/vercel/community/assets/62372737/6b3d815c-8063-4e2f-8d01-ebd675276a97)

### Steps to Reproduce

import repo in vercel deployement 
https://github.com/hrutik7/Emma/tree/master
and start deployment

**Comment by amyegan** [comment]
Hey @hrutik7. If you run [`vercel build`](https://vercel.com/docs/cli/build#) from your local development environment, do you get the same error? If so, you can try running it again with the  [`--debug`](https://vercel.com/docs/cli/global-options#debug) option to log extra info that can help identify the source of the problem.

If the build is successful on local, try to [deploy the prebuilt site to Vercel](https://vercel.com/docs/cli/deploying-from-cli#deploying-from-local-build-prebuilt) using `vercel deploy --prebuilt`. If that works, then I suspect a missing dependency is causing the problem.

**Reply by hrutik7** [reply]
hey @amyegan I fiexd the build and lint errors but after deploying i am getting edge function error
![image](https://github.com/vercel/community/assets/62372737/94d6f98f-c452-4738-afb3-49b250c02436)

**Reply by hrutik7** [reply, marked as answer]
error was about the casing once i rename the file name in kebab case the error is no more
